internal/assertions/assertions.go

```diff
@@ -162,6 +162,23 @@ func ErrorIs(err error, target error) (s string) {
 	return
 }
 
+func ErrorAs[E error, Target *E](err error, target Target) (s string) {
+	if err == nil {
+		s = "expected error; got nil\n"
+		return
+	}
+	if target == nil {
+		s = "expected target not to be nil"
+		return
+	}
+	if !errors.As(err, target) {
+		s = "expected errors.As match\n"
+		s += bullet(" error: %v\n", err)
```

Comment:
```suggestion
		s += bullet(" got: %v\n", err)
```

Both are errors, it would be clearer

But this wording was also used with ErrorIs, so either changing both or none